.github/workflows/pull-request.yml

```
@@ -9,482 +9,475 @@
+        - name: Checkout the Repository
+          uses: actions/checkout@v4
+        - name: Set up Node.js
+          uses: actions/setup-node@v4
+          with:
+            node-version: 20.x
+        - name: Install dependencies
+          run: npm install
+        - name: Clone API Repository
+          run: >
+            # Retrieve the complete branch name directly from the GitHub context
+  
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)